src/instructlab/sdg/flows/simple_freeform_skills.yaml

```diff
@@ -0,0 +1,13 @@
+version: "1.0"
+block_configs:
+  - block_type: LLMBlock
+    block_config:
+      block_name: gen_skill_freeform
+      config_path: configs/skills/simple_generate_qa_freeform.yaml
+      output_cols:
+        - output
+    gen_kwargs:
```

Comment:
When converting the 3 `simple` flows to yaml, there's an `n` value that got lost from `gen_kwargs` here

```
                "gen_kwargs": {
                    "max_tokens": 2048,
                    "temperature": 0.7,
                    "n": self.ctx.num_instructions_to_generate,
                },
```

you mentioned that the fix is setting it as a default. in llmblock

```diff
--- a/src/instructlab/sdg/llmblock.py
+++ b/src/instructlab/sdg/llmblock.py
@@ -60,6 +60,7 @@ class LLMBlock(Block):
         self.parser_cleanup_tags = parser_kwargs.get("parser_cleanup_tags", None)
         self.defaults = {
             "model": self.ctx.model_id,
+            "n": self.ctx.num_instructions_to_generate,
             "temperature": 0,
             "max_tokens": 12000,
         }
```